pkg/collector/corechecks/networkpath/networkpath.go

```diff
@@ -0,0 +1,122 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package networkpath defines the agent corecheck for
+// the Network Path integration
+package networkpath
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/DataDog/datadog-agent/comp/forwarder/eventplatform"
+	"github.com/DataDog/datadog-agent/pkg/aggregator/sender"
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/DataDog/datadog-agent/pkg/collector/check"
+	core "github.com/DataDog/datadog-agent/pkg/collector/corechecks"
+	"github.com/DataDog/datadog-agent/pkg/networkpath/traceroute"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/DataDog/datadog-agent/pkg/util/optional"
+)
+
+// CheckName defines the name of the
+// Network Path check
+const CheckName = "network_path"
+
+// Check doesn't need additional fields
+type Check struct {
+	core.CheckBase
+	config        *CheckConfig
+	lastCheckTime time.Time
+}
+
+// Program constants and default values
+const (
+	DefaultSourcePort   = 12345
+	DefaultDestPort     = 33434
+	DefaultNumPaths     = 10
+	DefaultMinTTL       = 1
+	DefaultMaxTTL       = 30
+	DefaultDelay        = 50 //msec
+	DefaultReadTimeout  = 3 * time.Second
+	DefaultOutputFormat = "json"
+)
```

Comment:
```suggestion
```

Looks like those are not used and duplicated in pkg/networkpath/traceroute/runner.go